tck/src/main/java/com/github/bsideup/liiklus/positions/PositionsStorageTests.java

```diff
@@ -0,0 +1,117 @@
+package com.github.bsideup.liiklus.positions;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import reactor.core.publisher.Flux;
+
+import java.time.Duration;
+import java.util.List;
+import java.util.Map;
+import java.util.UUID;
+
+import static java.util.Arrays.asList;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public interface PositionsStorageTests extends PositionsStorageTestSupport {
+
+    @Test
+    @DisplayName("Should save new position")
+    default void shouldSavePosition() {
+        GroupId groupId = GroupId.ofString(UUID.randomUUID().toString());
+
+        Map<Integer, Long> positions = asDeferMono(() -> getStorage().update(getTopic(), groupId, 2, 2))
+                .then(
+                        asDeferMono(() -> getStorage().findAll(getTopic(), groupId))
+                )
+                .block(Duration.ofSeconds(10));
+
+        assertThat(positions).containsEntry(2, 2L);
+    }
+
+
+    @Test
+    @DisplayName("Should update existing position")
+    default void shouldUpdatePosition() {
+        GroupId groupId = GroupId.ofString(UUID.randomUUID().toString());
+
+        Map<Integer, Long> positions = Flux
+                .concat(
+                        asDeferMono(() -> getStorage().update(getTopic(), groupId, 2, 2)),
+                        asDeferMono(() -> getStorage().update(getTopic(), groupId, 2, 3))
+                )
+                .then(
+                        asDeferMono(() -> getStorage().findAll(getTopic(), groupId))
+                )
+                .block(Duration.ofSeconds(10));
+
+        assertThat(positions)
+                .containsKeys(2)
+                .containsEntry(2, 3L);
+    }
+
+    @Test
+    @DisplayName("Should return all groups and topics")
+    default void shouldReturnMultipleGroups() {
+        GroupId groupId = GroupId.ofString(UUID.randomUUID().toString());
+        GroupId groupId2 = GroupId.ofString(UUID.randomUUID().toString());
+
+        List<PositionsStorage.Positions> positions = Flux
+                .concat(
+                        asDeferMono(() -> getStorage().update(getTopic(), groupId, 2, 2)),
+                        asDeferMono(() -> getStorage().update(getTopic(), groupId2, 2, 3)),
+                        asDeferMono(() -> getStorage().update(getTopic() + 1, groupId2, 2, 4)),
```

Comment:
`+1` looks weird here, would be better to extract a variable and use a random topic name instead.
Also, probably makes sense to not have `getTopic()` method at all, but use `val topic = UUID.randomUUID().toString()` in every test